### PR TITLE
[FIX] fix string empty

### PR DESCRIPTION
**Pull request by Jim8y**
This pr fix the string.Empty. Previously it returns "NULL" cause the compiler has filed to process this syntax, which actually should return "".

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs [hunk 1]
```diff
@@ -55,6 +55,16 @@ private void ConvertMemberAccessExpression(SemanticModel model, MemberAccessExpr
                 }
                 else if (field.IsStatic)
                 {
+                    // Have to process the string.Empty specially since it has no AssociatedSymbol
+                    // thus will return directly without this if check.
+                    if (field.ContainingType.ToString() == "string" && field.Name == "Empty")
+                    {
+                        // preInitialize?.Invoke();
```
Comment (Jim8y) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs:62]:
```suggestion
```

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs [hunk 2]
```diff
@@ -55,6 +55,15 @@ private void ConvertMemberAccessExpression(SemanticModel model, MemberAccessExpr
                 }
                 else if (field.IsStatic)
                 {
+                    // Have to process the string.Empty specially since it has no AssociatedSymbol
+                    // thus will return directly without this if check.
+                    if (field.ContainingType.ToString() == "string" && field.Name == "Empty")
+                    {
+                        Push(string.Empty);
+                        // postInitialize?.Invoke();
```
Comment (Jim8y) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs:63]:
```suggestion
```

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs [hunk 3]
```diff
@@ -55,6 +55,14 @@ private void ConvertMemberAccessExpression(SemanticModel model, MemberAccessExpr
                 }
                 else if (field.IsStatic)
                 {
+                    // Have to process the string.Empty specially since it has no AssociatedSymbol
+                    // thus will return directly without this if check.
+                    if (field.ContainingType.ToString() == "string" && field.Name == "Empty")
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs:60]: lowercase comparation and without the namespace?

Comment (Jim8y) [reply src/Neo.Compiler.CSharp/MethodConvert/Expression/MemberExpression.cs:60]: exactly, this is a built in static field of string,, will never change.

**Review by shargon — APPROVED** [review]

**Pull request merged**